src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/tests/latest/test_dls_commands.py

```diff
@@ -366,6 +368,32 @@ def test_dls_account_mgmt(self, resource_group):
             self.check('type(@)', 'array'),
             self.check('length(@)', 0),
         ])
+
+        # test virtual network rule crud
+        self.kwargs.update({
+            'vnet': 'lewuVNET',
+            'subnet_id': '/subscriptions/9e1f0ab2-2f85-49de-9677-9da6f829b914/resourceGroups/lewu-rg/providers/Microsoft.Network/virtualNetworks/lewuVNET/subnets/default',
+            'updated_subnet_id': '/subscriptions/9e1f0ab2-2f85-49de-9677-9da6f829b914/resourceGroups/lewu-rg/providers/Microsoft.Network/virtualNetworks/lewuVNET/subnets/updatedSubnetId'
+        })
+        self.cmd('dls account network-rule create -g {rg} --account {dls} --virtual-network-rule-name {vnet} --subnet {subnet_id}')
+        self.cmd('dls account network-rule show -g {rg} --account {dls} --virtual-network-rule-name {vnet}', checks=[
```

Comment:
`--virtual-network-rule-name` should be aliased to `--name/-n` for all commands within the `dls account network-rule` group.